test-data/unit/check-inference.test

```diff
@@ -420,7 +420,8 @@ class A: pass
 a = None # type: A
 
 def ff() -> None:
-    x = f() # E: Need type annotation for variable
+    x = f()
+    reveal_type(x)
```

Comment:
There's no output from `reveal_type`. The desired behavior would probably be to require an annotation for `x`.